src/functionalTest/resources/features/F-205 - Validate NoC Questions/205.12/205.12_Get_Organisation_Identifier_For_Dil.td.json

```diff
@@ -0,0 +1,15 @@
+{
+  "_guid_": "205.12_Get_Organisation_Identifier_For_Dil",
+  "_extends_": "Get_Organisation_Identifier__Base",
+
+  "specs": [
+    "As a prerequisite"
+  ],
+
+  "users": {
+    "invokingUser": {
+      "_extends_": "ACA_Users_Dil"
+    }
+  }
+
+}
```

Comment:
not sure if this lack of new line at end of file is an actual problem ?

Reply:
Resolved in 6339325 👍